components/DelegatingWidget/InputBox.tsx

```diff
@@ -13,10 +13,15 @@ import {
 import { fromWei } from "@lib/utils";
 import { ConsoleView } from "react-device-detect";
 
+interface Treasury {
+  treasuryRewardCutRate: number;
+}
+
 interface Props {
   transcoder: NonNullable<AccountQueryResult["data"]>["transcoder"];
   delegator?: NonNullable<AccountQueryResult["data"]>["delegator"];
   protocol: NonNullable<AccountQueryResult["data"]>["protocol"];
+  treasury: Treasury;
```

Comment:
The data type used here could benefit from improvement. I have now switched to using an object to maintain consistency with the other properties in this function.